backend/settings.py

```diff
@@ -202,12 +202,48 @@
 }
 
 TINYMCE_DEFAULT_CONFIG = {
-        'menubar': True,
-        "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
-        "toolbar": "formatselect | bold italic backcolor | "
-            "bullist numlist outdent indent | removeformat | code | help",
-        "default_link_target": "_blank",
-    }
+    'menubar': True,
+    "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
+    "toolbar": "bold italic backcolor | "
+        "bullist numlist outdent indent | removeformat | code | help",
+    "default_link_target": "_blank",
+    "invalid_elements": "h1, h2, h3, h4, h5, h6",
+    "invalid_styles": "color, font-family, background-color",
+
+    "style_formats": [
+        { 
+            "title": 'Inline', 
+            "items": [
+                { "title": 'Bold', "format": 'bold' },
+                { "title": 'Italic', "format": 'italic' },
+                { "title": 'Underline', "format": 'underline' },
+                { "title": 'Strikethrough', "format": 'strikethrough' },
+                { "title": 'Superscript', "format": 'superscript' },
+                { "title": 'Subscript', "format": 'subscript' },
+                { "title": 'Code', "format": 'code' }
+            ]
+        },
+        { 
+            "title": 'Blocks',  
+            "items": [
+                { "title": 'Paragraph', "format": 'p' },
+                { "title": 'Blockquote', "format": 'blockquote' },
+                { "title": 'Div', "format": 'div' },
+                { "title": 'Pre', "format": 'pre' }
+            ]
+        },
+        { 
+            "title": 'Align',  
+            "items": [
+                { "title": 'Left', "format": 'alignleft' },
+                { "title": 'Center', "format": 'aligncenter' },
+                { "title": 'Right', "format": 'alignright' },
+                { "title": 'Justify', "format": 'alignjustify' }
+            ]
+        }
+    ],
+    "block_formats": "Paragraph=p",
```

Comment:
```suggestion
    "block_formats": "Paragraph=p",
    "removed_menuitems": "fontformats fontsizes lineheight forecolor backcolor",
```

I think we we'll want to add this line. It seems to remove hide a lot of the settings we disallowed. Happy to show you how I found this if you'd like. I looked at this setting (https://www.tiny.cloud/docs/configure/editor-appearance/#removed_menuitems) and the menu setting (to determine item names) in the version 5 docs (what `django-tinymce` seems to be using).

Reply:
Thanks for the information! Line added.